packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -796,11 +794,11 @@ class RenderListWheelViewport
 
   /// Paints all children visible in the current viewport.
   void _paintVisibleChildren(PaintingContext context, Offset offset) {
-    RenderBox childToPaint = firstChild;
-    ListWheelParentData childParentData = childToPaint?.parentData as ListWheelParentData;
+    RenderBox? childToPaint = firstChild;
+    ListWheelParentData? childParentData = childToPaint?.parentData as ListWheelParentData?;
 
     while (childParentData != null) {
-      _paintTransformedChild(childToPaint, context, offset, childParentData.offset);
+      _paintTransformedChild(childToPaint!, context, offset, childParentData.offset);
       childToPaint = childAfter(childToPaint);
       childParentData = childToPaint?.parentData as ListWheelParentData;
     }
```

Comment:
if you move the setting of childParentData to inside the loop, and change the loop to be based on `childToPaint != null` rather than the childParentData, then you can make childParentData non-nullable with no change in semantics

Reply:
Acutally a `?` was missing on the `as ` line.

Reply:
right but my point still stands. :-)